server/src/main/java/org/elasticsearch/index/mapper/BinaryFieldMapper.java

```diff
@@ -226,7 +226,11 @@ public BytesRef binaryValue() {
             } catch (IOException e) {
                 throw new ElasticsearchException("Failed to get binary value", e);
             }
+        }
 
+        @Override
+        public String toString() {
```

Comment:
Is there a specific reason to implement toString() here? I quickly checked on KeywordFieldMapper and NumberFieldMapper and didn't find an implementation there either, just curious if this is something we generally would want to do or of this here is on special purpose.